client/my-sites/stats/checklist-banner/index.jsx

```diff
@@ -143,8 +132,15 @@ export class ChecklistBanner extends Component {
 	}
 
 	render() {
-		const { completed, total, translate, siteId } = this.props;
-		const task = this.getNextTask();
+		const { siteId, taskStatuses, translate } = this.props;
+		const total = tasks.length;
+		const completed = reduce(
+			tasks,
+			( count, { id, completed: taskComplete } ) =>
+				taskComplete || get( taskStatuses, [ id, 'completed' ] ) ? count + 1 : count,
```

Comment:
Worth turning 

```jsx
( { id, completed } ) => ! completed && ! get( this.props.taskStatuses, [ id, 'completed' ] )
```

into a method since we're using it in multiple places?

Reply:
This only exists in one place in the banner now, so we can leave it. The banner, ideally, will be implemented in another way eventually.

The other usage was removed by 479f892c1f, after I realized since we're not using connected `Tasks` anymore, we can _always_ infer checklist completion based on children as they should have a `completed` prop.

Reply:
> The other usage was removed by 479f892c1f, after I realized since we're not using connected `Tasks` anymore, we can _always_ infer checklist completion based on children as they should have a `completed` prop.

This is _very_ awesome!